<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsTicket</class>
 <widget class="QDialog" name="DetailsTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeader">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>EasyTicket</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitre">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre ticket</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="zoneMessages">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>511</width>
     <height>301</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>509</width>
      <height>299</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>511</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="zoneMessagesLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>59</y>
     <width>133</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="infosLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>50</number>
    </property>
    <item>
     <widget class="QLabel" name="labelID">
      <property name="text">
       <string>ID
0000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelStatut">
      <property name="text">
       <string>Statut
Ouvert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDate">
      <property name="text">
       <string>Créé le
01/12/2022</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCategorie">
      <property name="text">
       <string>Catégorie
Question</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelLogiciel">
      <property name="text">
       <string>Logiciel
Twisk</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPrisEnCharge">
      <property name="text">
       <string>Pris en charge par
Jean Dupond
01/12/2022</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boutonCloturer">
      <property name="text">
       <string>Clôturer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DetailsTicket</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DetailsTicket</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
